specification/scvmm/ScVmm.Management/client.tsp

```diff
@@ -3,4 +3,4 @@ import "@azure-tools/typespec-client-generator-core";
 
 using Azure.ClientGenerator.Core;
 
-@@clientName(Microsoft.ScVmm, "ScVmmMgmtClient", "python");
+@@clientName(Microsoft.ScVmm, "ScVmmMgmt", "python");
```

Comment:
`ScVmmMgmtClient` and `ScVmmMgmt` are both OK so no need for this PR.

Reply:
got it, closed